libs/libarchfpga/src/arch_util.cpp

```diff
@@ -616,6 +642,35 @@ void alloc_and_load_default_child_for_pb_type(t_pb_type* pb_type,
             copy->ports[i].port_power->buffer_type = POWER_BUFFER_TYPE_NONE;
         }
     }
+    // Special case for latch - fill secondary data fields
```

Comment:
Need a comment here or a reference to the header file to say what these secondary fields are for and how they're set (all copies of the latch ports etc.?  Why do they exist?)

Reply:
Linked to `t_pb_type` definition